geode-core/src/distributedTest/java/org/apache/geode/distributed/GrantorFailoverDUnitTest.java

```diff
@@ -180,8 +185,10 @@ public void lockRecoveryAfterGrantorDies() throws Exception {
   private static InternalDistributedMember assertIsElderAndGetId() {
     DistributionManager distributionManager =
         ClusterStartupRule.getCache().getInternalDistributedSystem().getDistributionManager();
-    Awaitility.await("Wait to be elder").atMost(1, TimeUnit.SECONDS)
-        .untilAsserted(() -> assertThat(distributionManager.isElder()).isTrue());
+    Awaitility.await("Wait to be elder").atMost(Duration.ONE_SECOND)
+        .conditionEvaluationListener((condition) -> logger
+            .info("Helena and Dale: View Members " + distributionManager.getViewMembers()))
```

Comment:
It looks like you're putting easily identifiable information in the log message to aid in debugging. I would suggest referencing the JIRA bug number here instead of listing individulal's names.